### PR TITLE
Update swagger.ts

### PR DESCRIPTION
**Pull request by mnholtz**
## What does this PR do?

- Part of https://github.com/pixiebrix/pixiebrix-app/pull/1703 (needed for ci to clear on this PR)
- This updates `swagger.ts`, mainly to integrate type changes to the Me type

## Reviewer Tips

n/a

## Discussion

n/a

## Demo

n/a

## Future Work

n/a

## Team Coordination

_Leave all that are relevant and check off as completed_

- [ ] This PR requires security review
- [ ] This PR introduces a new library: double check it's MIT/Apache2/permissively licensed
- [ ] This PR requires a node/npm version update: let the team know on #engineering
- [ ] This PR requires a documentation change (link to old docs)
- [ ] This PR requires a tutorial update (link to old tutorials)
- [ ] This PR requires a feature flag
- [ ] This PR requires a environment variable change

## Checklist

- [ ] Add tests
- [ ] Designate a primary reviewer

### CODE REVIEW

File: src/types/contract.ts [hunk 1]
```diff
@@ -220,7 +220,7 @@ export type RemoteResponse<T = unknown> = Pick<
 // Can't use Required. For blueprint_version, etc. the backend expects the property to be excluded or to have a value
 export type ErrorItem = Except<
   components["schemas"]["ErrorItem"],
-  "id" | "user" | "user_extension"
```
Comment (mnholtz) [inline src/types/contract.ts:223]: Is this what was intended?

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@pixiebrix/extension",
-  "version": "1.6.3-alpha.1",
```
Comment (twschiller) [inline package.json:3]: Let's not change this. The extension main branch we always keep at alpha.1 and the we version the release branches we cut off of main

File: src/types/contract.ts (hunk 1 above)
Comment (twschiller) [reply src/types/contract.ts:223]: > Is this what was intended?

What is "this"? Removing "id" from the object? Yes, the id is a surrogate key that doesn't appear in request/response payloads

cc @johnnymetz

File: src/types/swagger.ts [hunk 3]
```diff
@@ -989,6 +1006,12 @@ export interface components {
       /** @description True if the account is an organization API service account */
       service_account?: boolean;
       flags?: string;
+      partner?: {
```
Comment (twschiller) [inline src/types/swagger.ts:1009]: 👍

**Review by twschiller — APPROVED** [review]
@mnholtz LGTM, see comment about not changing the version number

File: package.json (hunk 2 above)
Comment (mnholtz) [reply package.json:3]: @twschiller does he version number get bumped automatically somewhere/somehow?

Comment (twschiller) [reply package.json:3]: > @twschiller does he version number get bumped automatically somewhere/somehow?

It's manual currently. When I create the first beta release branch, I make a commit to update the patch number of main

So when I create a branch for 1.6.3-beta.1, I'll bump main up to 1.6.4-alpha.1

File: src/types/contract.ts (hunk 1 above)
Comment (johnnymetz) [reply src/types/contract.ts:223]: Correct, `id` should not be in the request payload

**Pull request merged**